### DISCUSSION TITLE
How to Hide Assistant Builder?

### DISCUSSION THREAD

**Discussion by afunworm** [danny-avila/LibreChat#3385] [Q&A]
Hello.

I am really excited to share my custom trained AI with my co-workers at work, but then I realized the "Assistant Builder" was there, meaning anyone would be able to alter the edit the Assistant at ease. Is there any way to prevent that from happening? Maybe even just hiding them using CSS would working. I'd like people to use the Assistant, without the ability to modify it.

Thank you.

**Comment by fuegovic** [comment]
You can disable the assistants builder:

https://www.librechat.ai/docs/configuration/librechat_yaml/object_structure/assistants_endpoint#disablebuilder
